course/lesson_11/README.md

```diff
@@ -1,7 +1,8 @@
-### Lesson 11 - Organizing Files
+### Lesson 11 - argparse  + math + requests
```

Comment:
wprowadzamy requests?

math i argparse bym zrobił, trzeba dodać zadanka

Reply:
request to moja propozycja. Bedą musieli sami zainstalowac zewnetrza paczke i ogarnąć jej API.
a requests to jedna z popularnych i bardzo przydatnych paczek. W automate jest w Chapter 11 – Web Scraping.

Reply:
ja bym to przesunął na następny etap kursu i tam omówił tez parsowanie jsona, zaglądanie do nagłówków http, itp